binderhub/repoproviders.py

```diff
@@ -4,6 +4,8 @@
 Subclass the base class, ``RepoProvider``, to support different version
 control services and providers.
 
+Note: When adding a new repo provider, add it to the allowed values for
+      repo providers in event-schemas/launch.json.
```

Comment:
Can we replace this with a test that looks at all classes in the repo provider module and checks that their name is in the list?

Reply:
@betatim done!